dev-manuals/gamification/Suggestions.md

```diff
@@ -10,7 +10,7 @@ We differentiate between the suggestions inside a course and the suggestions bet
 
 A content can be a suggested content, if
 - the chapter is unlocked for the user
-- the content is unlocked for the user, i.e, he completed the required predecessors of the learning path aka section
+- the content is unlocked for the user, i.e. he completed the required predecessors of the learning path aka section
```

Comment:
This part is not really up to date with the actual implementation, but you don't need to do this in this PR

Reply:
That's okay. I just ran a language tool over this file and corrected any errors. I also added some more recent implementation details in the content-service.md under "suggestions" for this